sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -120,6 +120,12 @@ public partial class MatchConditions
         public Azure.ETag? IfMatch { get { throw null; } set { } }
         public Azure.ETag? IfNoneMatch { get { throw null; } set { } }
     }
+    public static partial class ModelSerializer
```

Comment:
Should this go in the Azure.Core.Serialization namespace?

Reply:
It probably will go in there if we need up going this way.  I will move them all now just in case.